src/backend/api/Fusion.Resources.Api/Controllers/Requests/InternalRequestsController.cs

```diff
@@ -691,6 +691,39 @@ public async Task<ActionResult<ApiResourceAllocationRequest>> ApproveProjectAllo
             return new ApiResourceAllocationRequest(result!);
         }
 
+        [HttpDelete("/resources/requests/internal/{requestId}/workflow")]
+        [HttpDelete("/departments/{departmentString}/resources/requests/{requestId}/workflow")]
+        public async Task<ActionResult> ResetWorkflow(Guid requestId, string? departmentString)
+        {
+            var requestItem = await DispatchAsync(new GetResourceAllocationRequestItem(requestId));
+
+            if (requestItem == null)
+                return ApiErrors.NotFound("Could not locate request", $"{requestId}");
+
+            #region Authorization
+
+            var authResult = await Request.RequireAuthorizationAsync(r =>
+            {
+                r.AlwaysAccessWhen().FullControl().FullControlInternal();
+                r.AnyOf(or =>
+                {
+                    if (!string.IsNullOrEmpty(requestItem.AssignedDepartment))
+                        or.BeResourceOwner(new DepartmentPath(requestItem.AssignedDepartment).Parent(), includeParents: false, includeDescendants: true);
+                    else
+                        or.BeResourceOwner();
+                });
+            });
+
+            if (authResult.Unauthorized)
+                return authResult.CreateForbiddenResponse();
+
+            #endregion
+
+            var deleted = await DispatchAsync(new ResetWorkflow(requestId));
+            if (deleted) return NoContent();
+            else return NotFound();
```

Comment:
Kind of ambigious. Could interpret this as the request does not exist, not the workflow. 
It could go either way if 404 is relevant, just delete it if it exists. In essense what is done here is setting a property to null on the request. 
In the least if you want to return a not found, a message should be included.